### PR TITLE
Docs: Update contributing guidelines

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR changes some of the docs contributing guidelines considering that the old docs site (built with mkdocs) is going away soon (#8756). I removed information about running the docs locally, updated some links, and then did a general grammar/copyedit run through the document.

This also fixes a pesky broken link that I couldn't fix correctly otherwise, and renames a few mentions of **docs.netdata.cloud**.

##### Component Name

docs

### CODE REVIEW

**Comment by joelhans** [comment]
@zack-shoylev No pressure to review, but you might find these documents valuable as part of your community efforts, and for yourself!

File: docs/contributing/style-guide.md [hunk 1]
```diff
@@ -21,9 +22,9 @@ products.
 ## Welcome
 
 Proper documentation is essential to the success of any open-source project. Netdata is no different. The health of our
-monitoring agent, and the community it's created, depends on this effort.
+monitoring agent and its community depends on this effort.
```
Comment (zack-shoylev) [inline docs/contributing/style-guide.md:25]: our community

File: docs/contributing/style-guide.md [hunk 2]
```diff
@@ -38,8 +39,8 @@ following principles:
 -   Conciseness
 -   Accessibility
 
-These principles will make documentation better for everyone who wants to use Netdata, whether they're a beginner or an
-expert.
+These principles will make the documentation better for everyone who wants to use Netdata, whether they’re a beginner or
```
Comment (zack-shoylev) [inline docs/contributing/style-guide.md:42]: shouldn't this be ' after all

File: docs/contributing/style-guide.md [hunk 3]
```diff
@@ -135,10 +136,10 @@ one-line installer requires fewer steps than manually installing from source."
 ### Avoid slang, metaphors, and jargon
 
 A particular word, phrase, or metaphor you're familiar with might not translate well to the other cultures featured
-among Netdata's global community. It's recommended you avoid slang or colloquialisms in your writing.
+among Netdata's global community. We recommended you avoid slang or colloquialisms in your writing.
 
-If you must use industry jargon, such as "white-box monitoring," in a document, be sure to define the term as clearly
-and concisely as you can.
+If you must use industry jargon, such as "white-box monitoring," in a document, define the term as clearly and concisely
+as you can.
```
Comment (zack-shoylev) [inline docs/contributing/style-guide.md:143]: Let's add something like this (as a section even): No usage of abbreviations that have not been defined before. I may be the first one to break this rule but it should still be in the rules...

Comment (zack-shoylev) [reply docs/contributing/style-guide.md:143]: We can move the abbreviation section up here maybe. Let me know

**Pull request merged**